syntax = "proto3";

import "proto/common_vfl.proto";

package vfl;

service StatusService {
  rpc GetStatus(StatusRequest) returns (StatusResponse);
}
